src/main/java/com/minecolonies/api/util/ItemStackUtils.java

```diff
@@ -1152,5 +1006,63 @@ public static void consumeFood(final ItemStack foodStack, final AbstractEntityCi
         }
         citizenData.markDirty(60);
     }
+
+    /**
+     * Get the tool level for vanilla tools.
+     *
+     * @param toolId The type of vanilla tool
+     * @param itemStack The item stack to check
+     * @return The tool level
+     */
+    public static int vanillaToolLevel(final String toolId, ItemStack itemStack) {
```

Comment:
Considering these methods are applicable to the builders inside of the initializer class and don't really have a usage outside there, please move.

Reply:
Done.